app/src/main/res/xml/preferences_user_interface.xml

```diff
@@ -29,6 +29,13 @@
                 android:key="prefDrawerFeedIndicator"
                 android:summary="@string/pref_nav_drawer_feed_counter_sum"
                 android:defaultValue="1"/>
+        <ListPreference
+            android:entryValues="@array/nav_drawer_feed_filter_values"
+            android:entries="@array/nav_drawer_feed_filter_options"
+            android:title="@string/pref_filter_feed_title"
+            android:key="prefFeedFilter"
+            android:summary="@string/pref_filter_feed_sum"
+            android:defaultValue="0"/>
```

Comment:
I would prefer using a normal `Preference` here and showing the dialog on click. This makes sure that the settings are always presented consistently.

Now we have 3 settings related to subscriptions (order, counter, filter). I think this would look great in a `PreferenceCategory` called "Subscriptions" (just like "Appearance" category above).

Please make sure to use the same indention as the rest of the file.